create_database.py

```diff
@@ -0,0 +1,136 @@
+    """
+    assert len(data_package) == 3, "Data packages consist of 3 strings of information."
+    given_name = name.get_given_name_from_label(data_package[0].lower())
+    family_name = name.get_family_name_from_label(data_package[1].lower())
+    user_cars = car.get_from_label(data_package[2])
+
+    if given_name and family_name and user_cars:
+        return (given_name,
+                family_name,
+                user_cars,)
+    else:
+        return None
+
+def prep_data(data_objects):
+    """Take in a list of data_objects in the format:
+    [('<given_name>', '<family_name>', ['<car>']),
+     ('<given_name>', '<family_name>', ['<car>', '<car>']),
+	 ('<given_name>', '<family_name>', ['<car>'])...]
+    Use the data in each data_object to create a unique userID and two
+    lists of tuples.
+    Prepare the first list for the rows of data in the 'users table'.
+    Prepare the second list for the rows of data in the 'cars table'.
+    Return the lists.
+    """
+    user_data_rows = []
+    car_data_rows = []
+
+    for data_object in data_objects:
+        if data_object:
+            assert len(data_object) == 3, "Data object needs 3 elements."
+
+			# Prepare the data rows for the 'users table'.
+            given_name = data_object[0]
+            assert name.is_valid(given_name)
+            family_name = data_object[1]
+            assert name.is_valid(family_name)
+            id_number = user_id.create(given_name, family_name)
+            assert user_id.is_valid(id_number)
+            user_data_rows.append((id_number, given_name, family_name))
+
+            # Prepare the data rows for the 'cars table'.
+            cars = data_object[2]
+            assert car.are_valid(cars)
+            for each_car in cars:
+                car_data_rows.append((each_car, id_number))
+
+    return user_data_rows, car_data_rows
+
+def create_users_table(db_cursor, user_data_rows):
+    """Create a table of users with columns for userID, given_name,
+    and family_name. Insert the user_data_rows into the table.
+    """
+    # Create 'users table' in the given database.
+    db_cursor.execute("""
+        CREATE TABLE users(userID TEXT PRIMARY KEY,
+                           given_name TEXT,
+                           family_name TEXT)
+    """)
+
+    # Insert user data rows into the 'users table'.
+    for row in user_data_rows:
+        if row:
+            assert len(row) == 3, "Wrong amount of information in the row of data."
+
+            db_cursor.execute("""INSERT INTO users(userID, given_name, family_name)
+                              VALUES(?,?,?)""", row)
+
+def create_cars_table(db_cursor, car_data_rows):
+    """Create a table of cars with columns for car and owner. Insert
+    the car_data_rows into the table.
+    """
+    # Create 'cars table' in the given database.
+    db_cursor.execute("""
+        CREATE TABLE cars(car TEXT,
+                          owner TEXT)
```

Comment:
might want to use similar naming - call this user id rather than owner, or the other one owner rather than user id